docs/API_DESIGN.md

```diff
@@ -112,6 +123,22 @@ CLASS CreateOptions
   PUBLIC DATA type: []string?
   PUBLIC DATA issuance_date: datetime?
   PUBLIC DATA expiration_date: datetime?
+  PUBLIC DATA credential_status: CredentialStatus?
+```
+
+## StatusListCredential
+
+#### `StatusListCredential`
+
+```pseudocode!
+CLASS StatusListCredential IMPLEMENTS VerifiableCredential
+PUBLIC DATA status_purpose: string
+PUBLIC DATA credentials_to_disable: []VerifiableCredential
+
+CONSTRUCTOR create(issuer: Issuer, status_purpose: string, credentials_to_disable: []VerifiableCredential, options: CreateOptions?)
+
+METHOD update_credentials_to_disable(credentials_to_disable: []VerifiableCredential): StatusListCredential
```

Comment:
is this a setter function? or is it appending to the array? if it's the latter, you could do `append_to_credentials_to_disable`

is `StatusListCredential` a data model in a spec, or is this our data model? if it's ours I would go with `disabled_credentials` instead of `credentials_to_disable`

Reply:
It’s not append , it’s regenerating the entire list, a better name could be regenerate_*somethingsomething*

and disabled credentials rewording is good! I’ll use that

Reply:
word, if it's a setter function then `update_*` or `set_*` is good